benchmark_runner/common/oc/oc.py

```diff
@@ -457,32 +457,51 @@ def get_worker_nodes(self):
         """
         return self.run(fr""" {self.__cli} get nodes -l node-role.kubernetes.io/worker= -o jsonpath="{{range .items[*]}}{{.metadata.name}}{{'\n'}}{{end}}" """)
 
-    @staticmethod
     @typechecked
-    def check_node_status(nodes_list: list):
+    def wait_for_nodes_ready(self, wait_time: int = None, timeout: int = int(environment_variables.environment_variables_dict['timeout'])):
         """
-        This method check node status
-        @param nodes_list:
-        @return: True when all nodes in ready status
+        This method waits until all nodes are in 'Ready' status.
+        @param wait_time: wait time between each loop
+        @param timeout: Maximum wait time in seconds (default set in environment variables)
```

Comment:
Need to document that negative value means no timeout (which I think is a good thing to have).

Reply:
Add it to the document